backend/module/eCampLib/src/InputFilter/HtmlPurify.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace eCamp\Lib\InputFilter;
+
+use HTMLPurifier;
+use Laminas\Filter\AbstractFilter;
+
+class HtmlPurify extends AbstractFilter {
+    /** @var HTMLPurifier */
+    private $htmlPurifier;
+
+    public function __construct($options = null) {
+        $this->options = $options;
+    }
+
+    public function getHtmlPurifier() {
+        if ($this->htmlPurifier instanceof HTMLPurifier) {
+            return $this->htmlPurifier;
+        }
+        $this->htmlPurifier = new HTMLPurifier($this->options);
+
+        return $this->htmlPurifier;
```

Comment:
Könnte man das vereinfachen?
```suggestion
        if (!($this->htmlPurifier instanceof HTMLPurifier)) {
            $this->htmlPurifier = new HTMLPurifier($this->options);
        }
        return $this->htmlPurifier;
```

Reply:
fixed